lte/gateway/c/session_manager/ShardTracker.h

```diff
@@ -37,6 +37,18 @@ class ShardTracker {
    */
   bool remove_ue(const std::string imsi, const uint16_t shard_id);
 
+  /**
+   * Retrieve number of shards currently in shard tracker
+   * @return int representing number of shards that have been generated
+   */
+  uint16_t get_shard_list_size();
+
+  /**
+   * Retrieve a list of active shard ids
+   * @return vector<int> representing shard IDs of shards that have UEs
+   */
+  std::vector<int> get_active_shards();
```

Comment:
`std::vector< uint16_t >`?

Reply:
The issue is that cookies can be up to size uint32 on pipelined side, and these values will get mapped directly to cookies and type mismatch errors happen.

Reply:
oh interesting. Then why did we pick uint16_t on SessionD's side?

Reply:
I picked uint16 because I didn't think we wanted a lot of shards in concept. We want for shards to be relatively large in size capabale of holding a lot of UEs, and they'll help well in multiprocessing and smart polling.